querybook/server/datasources/query_execution.py

```diff
@@ -374,9 +374,9 @@ def export_statement_execution_result(
 
 
 @register("/query_execution/templated_query/", methods=["POST"])
-def get_templated_query(query: str, variables: Dict[str, str]):
+def get_templated_query(query: str, variables: Dict[str, str], engine_id: int = None):
```

Comment:
fixed to make it required (originally didn't make it required since the render query function can technically still work without the engine/metastore_loader if `latest_partition` isn't used in the query)

Reply:
it is still optional after the update, is it resolved?